crds/crd-coralogix-alert-set.yaml

```diff
@@ -78,6 +78,23 @@ spec:
                                 descprition: Sends resolve notification when the alert criteria that you defined is no longer met
                                 type: boolean
                                 default: true
+                              metricAlertPromqlParameters:
+                                description: Data for defining promql metric
+                                type: object
+                                properties:
+                                  promqlText:
+                                    type: string
+                                  arithmeticOperatorModifier:
+                                    type: integer
+                                    format: int32
+                                  sampleThresholdPercentage:
```

Comment:
What it means in the UI? cant really understand

Reply:
it should be `For over` input
<img width="525" alt="Screenshot 2022-06-20 at 23 25 54" src="https://user-images.githubusercontent.com/10402628/174679614-621aeb12-6a37-47fc-8bcc-7eb32bc99758.png">

Reply:
So can we rename it?